### DISCUSSION TITLE
Add documentation of limitations

### DISCUSSION THREAD

**Discussion by timsofteng** [d99kris/nchat#84] [Ideas]
Hello. It would be cool to see list of limitations in readme. Like some which is in roadmap and some which is impossible to implement. E.g voip, search and so on.
Thanks.

**Comment by d99kris** [comment]
Hi @timsofteng - I think that's a good idea 👍 

Let me take a look at this and get back.

**Reply by d99kris** [reply]
Will track addressing this under #85.

**Reply by d99kris** [reply]
The README has now been updated with new sections "Limitations" and "Roadmap".
